integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwSample.java

```diff
@@ -0,0 +1,473 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createRcuSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretForBaseImages;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getExternalServicePodName;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.DbUtils.getNewSuffixCount;
+import static oracle.weblogic.kubernetes.utils.FileUtils.replaceStringInFile;
+import static oracle.weblogic.kubernetes.utils.TestUtils.callWebAppAndWaitTillReady;
+import static oracle.weblogic.kubernetes.utils.TestUtils.getNextFreePort;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.apache.commons.io.FileUtils.copyDirectory;
+import static org.apache.commons.io.FileUtils.deleteDirectory;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests related to FMW domain samples.
+ */
+@DisplayName("Verify the JRF domain on pv sample using wlst")
+@IntegrationTest
+public class ItFmwSample {
+
+  private static String dbNamespace = null;
+  private static String domainNamespace = null;
+
+  private static final Path samplePath = Paths.get(ITTESTS_DIR, "../kubernetes/samples");
+  private static final Path tempSamplePath = Paths.get(WORK_DIR, "sample-testing");
+  private final String domainUid = "jrfsamplepv";
+
+  private static final String RCUSCHEMAPREFIX = "jrfsamplepv";
+  private static final String ORACLEDBURLPREFIX = "oracle-db.";
+  private static final String ORACLEDBSUFFIX = ".svc.cluster.local:1521/devpdb.k8s";
+  private static final String RCUSYSUSERNAME = "sys";
+  private static final String RCUSYSPASSWORD = "Oradoc_db1";
+  private static final String RCUSCHEMAUSERNAME = "myrcuuser";
+  private static final String RCUSCHEMAPASSWORD = "Oradoc_db1";
+  private static String dbUrl = null;
+  // in general the node port range has to be between 30,000 to 32,767
+  // to avoid port conflict because of the delay in using it, the port here
+  // starts with 30100
+  final int t3ChannelPort = getNextFreePort(30172, 32767);
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(10, MINUTES).await();
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Start DB service and create RCU schema.
+   * Assigns unique namespaces for operator and domains.
+   * Pull FMW image and Oracle DB image if running tests in Kind cluster.
```

Comment:
Why only on Kind cluster ?  Should the statement be generic as follows ...

Pull FMW image and Oracle DB image if needed.

Reply:
Done